source/resolve-conflicts.ts

```diff
@@ -10,12 +10,8 @@ declare module CodeMirror {
 	}
 }
 
-interface EditorLinkedDom extends HTMLElement {
-	CodeMirror: CodeMirrorInstance;
-}
-
-const select = document.querySelector.bind(document);
-const editor = select<EditorLinkedDom>('.CodeMirror')!.CodeMirror;
+const select: typeof document.querySelector = document.querySelector.bind(document);
```

Comment:
I just realized `select` is no longer used anywhere else, so you can just drop this altogether 😅

Reply:
Done.